g0v.network./polis-explorations.g0v.network.yml

```
@@ -0,0 +1,18 @@
+---
+polis-explorations:
```

If we want to redirect `explorations.polis.g0v.network` instead:

```suggestion
explorations:
```

Would also need to rename file
`g0v.network./polis-explorations.g0v.network.yml` => `g0v.network./explorations.polis.g0v.network.yml`